### PR TITLE
Convert strikethrough in Markdown

### PR DESCRIPTION
**Pull request by ellatrix**
## Description
This branch adds strikethrough support for Markdown conversion when pasting.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: test/integration/fixtures/markdown-in.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This is a heading with *italic*
```
Comment (tofumatt) [inline test/integration/fixtures/markdown-in.txt]: I know this is unrelated, but it's weird that this file is `.txt`. Just figured I'd point that out.

**Review by tofumatt — APPROVED** [review]
Looks good to me!

File: test/integration/fixtures/markdown-in.txt (hunk 1 above)
Comment (iseulde) [reply test/integration/fixtures/markdown-in.txt]: Why is it weird? :)

Comment (tofumatt) [reply test/integration/fixtures/markdown-in.txt]: It's a Markdown fixture file–I'd just expect the file extension to be either `.md` or `.markdown`, which are the usual extensions for a file written in Markdown. That way editors will syntax highlight it, etc.

Comment (iseulde) [reply test/integration/fixtures/markdown-in.txt]: Oh, this was to differentiate between HTML (.html) and plain text (.txt) input.

**Pull request merged**